pom.xml

```diff
@@ -32,7 +32,7 @@
     </parent>
 
     <artifactId>cloudbees-bitbucket-branch-source</artifactId>
-    <version>1.6-SNAPSHOT</version>
+    <version>1.6-unifio</version>
```

Comment:
This was just to differentiate our version with the official.  Again, the only file needed to change is BitbucketServerAPIClient.java.